src/dmd/opover.d

```diff
@@ -2089,3 +2057,26 @@ private bool matchParamsToOpApply(TypeFunction tf, Parameters* parameters, bool
     }
     return true;
 }
+
+/**
+ * Reverse relational operator, eg >= becomes <=
+ * Note this is not negation.
+ * Params:
+ *      op = comparison operator to negate
+ * Returns:
+ *      negate operator
```

Comment:
Params and Returns have been copy-pasted

Reply:
??

Reply:
"comparison operator to negate" & "negate operator"

This is not negation, this is reversing, according to the "Note".